### ISSUE TITLE
Error reporting unknown directive "strict_types " if whitespace used

### ISSUE THREAD

**Issue by Haehnchen** [kalessil/phpinspectionsea#388]
Missing whitespace trim in "strict_types " detection.

```
declare(strict_types=1); OK
declare(strict_types = 1); Reports error
```

**Comment by funivan** [comment]
This seems to be implemented in latest versions of PhpStorm. 
http://imagizer.imageshack.com/img922/6066/zs1WkC.png
PhpStorm 
2017.2 Public preview
version: 172.3317.9

**Comment by Haehnchen** [comment]
no its addressed by last phpinspectionsea update

**Comment by funivan** [comment]
Confirmed

**Comment by kalessil** [comment]
Ooops, fixed 2.3.8.1 coming shortly.
